### ISSUE TITLE
disable chat plugin

### ISSUE THREAD

**Issue by simonnygh** [facebookincubator/facebook-for-woocommerce#835]
Where is this setting? Please post a step-by-step instructions to disable the messenger chat plugin using settings, thank you!

**Comment by jieminz22** [comment]
## Automated Support Message

Hello,

Thanks for your interest in our plugin! Our developer team tries to provide personal support on GitHub whenever possible.

If you have a feature request or feedback for the developers, reply with `Feature Request` or `Feedback`.
If you are a developer or have a question for the developers related to code, reply with `Developer Question`.
If you are running ads with us, reply with `Advertiser`.

If you have encountered a bug or a problem with the plugin, keep reading for information that can help you resolve the issue yourself or provide us details we need to help you.

## Search Existing Issues

First, please [search our issues](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) to see if your problem has been reported before.
There may already be a workaround or a solution to your problem. You can use the [search bar](https://github.com/facebookincubator/facebook-for-woocommerce/issues?q=) near the top of our issues page. Be sure to search both _open_ and _closed_ issues.
If you do find a similar issue, leave a comment on that issue instead of creating a new one. Your comment helps us prioritize which existing issues are important and urgent.

## Common Resolution Methods

If you are experiencing a problem or bug and have not found any similar issue, please try the following:

* Upgrade to the latest version of the plugin
* Try to `Resync` products using the button on the plugin config page
* Reset the plugin by selecting `Manage Settings > Advanced Options > Delete` from the configuration page of the plugin and setting up again
* Try deactivating other plugins that you think may be interfering, then reset and set up again

## Necessary Information

If the above steps did not help, please provide the following information in your next comment in order for us to debug your problem:

* WooCommerce version
* WordPress version
* Plugin version (please note: we only support up-to-date versions of the plugin)
* A screenshot of where you see the problem (for example: if it is on Facebook, provide a screenshot of Facebook, if it is on WooCommerce, provide a screenshot of WooCommerce. You can provide multiple screenshots if the problem relates to different screens.)
* The URL of your website
* The URL of your Facebook Page
* The URL(s) of where you took your screenshot(s)
* If applicable, detailed steps you took in order to see the problem

Please include `Debug Ready` in your reply so our automated system can note your issue for manual review. Please be aware that due to a high volume of issues, we may not be able to respond immediately.

## Response Time

We prioritize issues with the tags `Advertiser` or `Multiple Reports`.
For issues that have the tags `Debug Ready`, `Feature Request` or `Developer Question`, we aim to respond within 30 days.

If you have not responded to our automated message with one of the above phrases, such as `Debug Ready`, we may close your issue after 30 days. If a `Feature Request` is inactive (without new requests, messages or pull requests) for 6 months, we may close it.

Please note that we are not always able to provide support for issues related to interaction from other plugins with this plugin or related to the Facebook advertising interface. To get support related to Facebook ads, try visiting the [Facebook Ads Help Center](https://www.facebook.com/business/help).

For more information on our response times, visit [Facebook Support on GitHub](https://github.com/facebookincubator/facebook-for-woocommerce/wiki/Facebook-Support-on-GitHub).

Thank you,
The Facebook Dev Team

**Comment by simonnygh** [comment]
debug ready

**Comment by simonnygh** [comment]
Sorry, which "Settings"? the WordPress settings found at wp-admin -> WooCommerce -> Settings -> Integration -> Manage Settings (under facebook)? If not, where is this "Settings" option/button?

**Comment by jieminz22** [comment]
It's in Manage Settings

**Comment by robertofrega** [comment]
I have the same problem here, but even though I followed the steps, the plugin is not chat is not being disabled.

I re-enabled it setting the custom colors and custom message, but after reloading the page, the widget kept its custom settings.

Of course I trying disabling it again, but to no effect.

Another thing that is happening (that may help solve the issue) is that the product feed is not syncing and keeps showing that message "Your API key is no longer valid. Please click "Settings > Advanced Options > Update Token". (Which for me is fine, since I was already syncing it through an XML feed anyway.)

I use the plugin mostly because of the automatic js tags (ViewContent, AddToCart, etc.), but if the chat keeps showing, I may have to disable it and tag manually via GTM. Which is a bother.

**Comment by beehavendesignz** [comment]
I too am having this same issue as freguinhas

I have followed all the steps to disable the chat but it will not disable.  

Also Another thing that is happening (that may help point to the issue) is that the product feed is not syncing and keeps showing that message "Your API key is no longer valid. Please click "Settings > Advanced Options > Update Token".

**Comment by cengizgoren** [comment]
Hi there.. if you want to disable the  Facebook Messenger you have to do the following steps.

Step 1: **Go to your Fanpage → Settings → Messenger Platform**. 

Step 2: On the 7th Tab left.. select and find your connected domain.. In the Whitelisted Domains, remove your domains of your website(s). 

Step 3: Then click Save to complete.

> Et voila.. the Messenger is gone !

i found the solution here.. 
[https://gretathemes.com/facebook-customer-chat/]

**Comment by lgfogle** [comment]
Hey there! 👋

Lindsey here, from the SkyVerge product team! We recently took over development and support for the Facebook for WooCommerce extension.

In our recent 1.10 releases, we've fixed many issues impacting the store connection, pixel tracking, and product sync. Could you please update to the latest version of Facebook for WooCommerce and see if that resolves this issue?

If not, could you please [contact our support team](https://woocommerce.com/my-account/create-a-ticket/) or open a new issue here? We'd be happy to help troubleshoot further with you.

Thank you!

**Comment by mitchellkrogza** [comment]
@lgfogle I've lodged so many support requests on Facebook for Messenger and they are all ignored. You've released 5 new versions over the past few weeks and not addressed the inability to disable Messenger. I had to had to force this by editing the wp_options table and setting the wc_facebook_enable_messenger to no. But revisiting the Messenger of your tab just re-enables it again. I have it disabled now and now avoid clicking on the Messenger tab in the plugin page but a simple on/off switch that actually works would be great. There are numerous other people complaining about this in the plugin support page on Wordpress but nobody seems to get any replies to any support requests?
